### PR TITLE
Update FSC

### PR DESCRIPTION
**Pull request by adecaro**
Testing FSC versions

### CODE REVIEW

File: tools/tools.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 	_ "github.com/gordonklaus/ineffassign"
 	_ "github.com/maxbrunsfeld/counterfeiter/v6"
 	_ "github.com/onsi/ginkgo/v2/ginkgo"
+	_ "github.com/securego/gosec/v2/cmd/gosec"
```
Comment (mbrandenburger) [inline tools/tools.go:18]: It seems this should be in a separate PR to enable its use via the makefile or even GH action.

**Review by mbrandenburger — COMMENTED** [review]
Did you forget to add the FSC update in `go.mod` as suggested by the title of this PR.

File: tools/tools.go (hunk 1 above)
Comment (adecaro) [reply tools/tools.go:18]: good catch, will remove

**Review by alexandrosfilios — APPROVED** [review]
LGTM
